tests/testthat/test-sparse.R

```diff
@@ -1,14 +1,9 @@
 context("sparse and dense comparisons")
+source("helpers.R")
```

Comment:
IIRC, `testthat` will automatically pre-load any `*.R` files starting with `helper_`

Reply:
Nice, I didn't know!

Reply:
Hm, I had to use a setup- file instead. For some reason the helper- file did not load properly for all the tests.